frontend/client/utils/markdown.ts

```diff
@@ -27,7 +27,6 @@ const converters: { [key in MARKDOWN_TYPE]: Showdown.Converter } = {
     ...sharedOptions,
     noHeaderId: true,
     headerLevelStart: 4,
-    // TODO: Find a way to disable images
```

Comment:
To confirm, these are removed right? Didn't see a note in your testing.

Reply:
Tested this out, looks like it works. Neat-o!

Reply:
The whitelist for REDUCED type doesn't include img.
https://github.com/grant-project/zcash-grant-system/blob/3050caf32952415f18ed0513365149eea1ea293e/frontend/client/utils/markdown.ts#L42